src/SFA.DAS.Admin.Roatp.Web/Controllers/TrainingProviderController.cs

```diff
@@ -0,0 +1,54 @@
+﻿using FluentValidation;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using SFA.DAS.Admin.Roatp.Web.Infrastructure;
+using SFA.DAS.Admin.Roatp.Web.Models;
+using SFA.DAS.Admin.Roatp.Web.Services;
+
+namespace SFA.DAS.Admin.Roatp.Web.Controllers;
+
+[Authorize(Roles = Roles.RoatpAdminTeam)]
+[Route("selectProvider", Name = RouteNames.SelectProvider)]
+public class TrainingProviderController(ISessionService _sessionService, IValidator<SelectTrainingProviderSubmitViewModel> _validator) : Controller
+{
+    [HttpGet]
+    public IActionResult Index()
+    {
+        SelectTrainingProviderViewModel model = new SelectTrainingProviderViewModel();
+
+        // THIS IS A TEMPORARY MEASURE TO ENABLE TESTING, REMOVED in CSP-2210
+        var savedOrganisation = _sessionService.Get<OrganisationSessionModel>(SessionKeys.EditOrganisation);
```

Comment:
I am against having anything temporary in the code. If I have to hand this over to someone else, it will be difficult.